docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -1,246 +1,211 @@
-a safe method of doing so for the specific edge case of a lazy masked integer
-array:
-
-.. doctest::
-
-    >>> values = np.zeros((73, 96), dtype=int)
-    >>> data =np.ma.masked_values(values, 0)
-    >>> print(data)
-    [[-- -- -- ..., -- -- --]
-     [-- -- -- ..., -- -- --]
-     [-- -- -- ..., -- -- --]
-     ...,
-     [-- -- -- ..., -- -- --]
-     [-- -- -- ..., -- -- --]
-     [-- -- -- ..., -- -- --]]
-    >>> new_cube = cube.copy(data=data)
-    >>> new_cube.has_lazy_data()
+    >>> points = aux_coord.points
+    >>> print aux_coord.has_lazy_points()
     False
-    >>> new_cube.data = new_cube.lazy_data()
-    >>> new_cube.has_lazy_data()
+    >>> print derived_coord.has_lazy_points()
     True
+    >>> print derived_coord.has_bounds()
+    True
+    >>> print derived_coord.has_lazy_bounds()
+    False
 
-This method is necessary as Dask is currently unable to handle masked arrays.
-Please refer to the Whitepaper for further details.
-
-
-Coordinate Arrays
------------------
-
-Cubes possess coordinate arrays as well as data arrays, so these also benefit
-from Dask's functionality, although there are some distinctions between how
-the different coordinate types are treated.
+.. note::
+    Printing a lazy :class:`~iris.coords.AuxCoord` will realise its points and bounds arrays!
 
-Auxiliary coordinates can now contain lazy arrays, so they will adhere to the
-same rules and behaviour as the data arrays.  Dimension coordinates, however,
-undergo monotonicity checks which cause the arrays to be realized upon
-construction, so they can only contain real arrays.
+Derived coordinates (also called aux factories) .
 
 
-Dask Processing Options
+Dask processing options
 -----------------------
-Dask applies some default values to certain aspects of the parallel processing
-that it offers with Iris. It is possible to change these values and override
-the defaults by using 'dask.set_options(option)' in your script.
-
-You can use this as a global variable if you wish to use your chosen option for
-the full length of the script, or you can use it with a context manager to
-control the span of the option.
-
-Here are some examples of the options that you may wish to change.
-
-You can set the number of threads on which to work like this:
-
-.. doctest::
 
-    >>> import dask
-    >>> from multiprocessing.pool import ThreadPool
-    >>> with dask.set_options(pool=ThreadPool(4)):
-    ...     x.compute()
-
-Multiple threads work well with heavy computation.
+As stated earlier in this user guide section, Iris uses dask to provide
+lazy data arrays for both Iris cubes and coordinates. Iris also uses dask
+functionality for processing deferred operations on lazy arrays.
 
+There are a wide range of dask processing options that can be adjusted to
+control how dask processes deferred operations on lazy arrays. You can make use
+of these dask processing options to control how lazy arrays within Iris are
+processed as well.
```

Comment:
I had to read these two sentences about three times before I understood the difference between them.  Maybe needs a bit of rewording.  Just a smidge.